### DISCUSSION TITLE
Separate build command for preview builds

### DISCUSSION THREAD

**Discussion by akshaybabloo** [vercel/community#5638] [Help]
### Summary

I am coming from Netlify and they have a way to run a completely different command for preview builds. Is it possible to do the same in Vercel? I am using `vercel.json` to configure the `buildCommand`.

### Example

Here is an example of Netlify configuration - https://github.com/akshaybabloo/gollahalli.com/blob/f129d2c8a06478c23f0ceec0c50a202ecf1be42f/netlify.toml#L16

### Steps to Reproduce

N/A

**Comment by Kopamed** [comment]
Did you find a solution to this?
